dashboard/src/t5gweb/taskmgr.py

```diff
@@ -129,4 +137,23 @@ def cache_data(data_type):
     elif data_type == 'escalations':
         libtelco5g.cache_escalations(cfg)
     else:
-        logging.warning("unkown data type")
\ No newline at end of file
+        logging.warning("unknown data type")
+
+@mgr.task
+def tag_bz():
+    
+    logging.warning("getting bugzillas")
+    bz_url = "bugzilla.redhat.com"
+    cfg = t5gweb.set_cfg()
+    bz_api = bugzilla.Bugzilla(bz_url, api_key=cfg['bz_key'])
+    bugs = libtelco5g.redis_get('bugs')
+    logging.warning("tagging bugzillas")
+    for case in bugs:
+        for bug in bugs[case]:
+            bz = bz_api.getbug(bug['bugzillaNumber'])
+            if "telco" not in bz.internal_whiteboard.lower():
```

Comment:
I'm wondering... can we get the internal whiteboard stuff when we cache the BZs? that way we don't need to call the bz API on every bug.

Reply:
Yeah, I think that would better! I just added the internal_whiteboard to the cache

Reply:
Awesome, thanks!